lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module Capybara
+        # Checks for if there is a more specific matcher offered by Capybara.
+        #
+        # @example
+        #
+        #   # bad
+        # expect(page).to have_css('button')
+        # expect(page).to have_css('button', exact_text: 'foo')
+        # expect(page).to have_css('button[disabled]', exact_text: 'foo')
+        # expect(page).to have_css('a')
+        # expect(page).to have_css('a.cls', exact_text: 'foo')
+        # expect(page).to have_css('table')
+        # expect(page).to have_css('table.cls')
```

Comment:
A more generic `have_selector` needs to be checked, too.

Reply:
2x much offences detected with `have_selector` included:
```
﻿﻿57497 files inspected, 167 offenses detected
```

Reply:
Oh! Added `have_selector`。